services/QuillLMS/.nvmrc

```diff
@@ -1 +1 @@
-lts/erbium
+lts/fermium
```

Comment:
To keep consistency with our CI, I would update the circleci script to reflect this upgrade:

https://github.com/empirical-org/Empirical-Core/blob/develop/.circleci/config.yml#L152
https://github.com/empirical-org/Empirical-Core/blob/develop/.circleci/config.yml#L313